docs_user/modules/proc_adopting-compute-services-to-the-data-plane.adoc

```diff
@@ -148,6 +148,29 @@ oc get secret nova-migration-ssh-key || oc create secret generic nova-migration-
 rm -f id*
 cd -
 ----
++
+
+. Set LIBVIRT_PASSWORD to match the existing {OpenStackShort} deployment password:
```

Comment:
This should be given a condition, like https://github.com/openstack-k8s-operators/data-plane-adoption/blob/main/docs_user/modules/proc_adopting-compute-services-to-the-data-plane.adoc?plain=1#L173